h2o-py/tests/testdir_algos/glm/pyunit_pubdev_6037_fractionalbinomial_pojo_mojo.py

```diff
@@ -0,0 +1,36 @@
+import sys, os
+sys.path.insert(1, "../../../")
+import h2o
+from tests import pyunit_utils
+
+def glm_ordinal_mojo_pojo():
+    params = set_params()   # set deeplearning model parameters
```

Comment:
```suggestion
    params = set_params()
```

Forgotten commentary ?